x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/enrich/10_basic.yml

```diff
@@ -13,8 +11,9 @@
                 type: keyword
               b:
                 type: keyword
-  - is_true: acknowledged
```

Comment:
Any reason for deleting this line, or was that an accident?

Reply:
Not an accident. `setup:` can't have test statements like `is_true:` -- if you look at the other `enrich/*.yml` test files, they all have a similar `indices.create` at the top, and none of them have `is_true: acknowledged`.